sources/packages/backend/libs/integrations/src/institution-integration/ece-integration/ece-files/ece-file-header.ts

```diff
@@ -0,0 +1,29 @@
+import {
+  DATE_FORMAT,
+  ECERequestFileLine,
+  ECE_SENT_TITLE,
+  RecordTypeCodes,
+  SPACE_FILLER,
+} from "../models/ece-integration.model";
+import { StringBuilder } from "@sims/utilities";
+
+const ORIGINATOR_CODE = "SSB";
+
+/**
+ * Header of a ECE request file.
+ * The documentation about it is available on the document
+ * 'SIMSSFAS - Institution File layouts In Analysis Folder'.
+ */
+export class ECEFileHeader implements ECERequestFileLine {
+  transactionCode: RecordTypeCodes;
```

Comment:
why is record type named as transaction code, Asked it because I was searching for record type inside `getFixedFormat()`

Reply:
Because it was copied and copied again from CRA where `transactionCode` makes sense.
I made similar comment here: https://github.com/bcgov/SIMS/pull/1641#discussion_r1062858721